app/views/items/_item.html.erb

```diff
@@ -1,9 +1,9 @@
 <div class="row">
   <div class="col-md-8">
-    <% @user.items.each do |item| %>
-      <div class='media', id="item-<%=item.id %>">
-        <font size="4.5"><%= item.name %></font><%= link_to "", item, method: :delete, remote: true, class: 'glyphicon glyphicon-ok' %>
+    <%# @items.each do |item| %>
```

Comment:
no reason to leave commented code.  If you look at this on GitHub it shoes what you deleted to the left